### ISSUE TITLE
How to configure a dashboard static range?

### ISSUE THREAD

**Issue by TheWiresharkGuy** [traggo/server#101]
**Have you read the documentation?**
- [ ] Yes, but it does not include related information regarding my question.
- [ ] Yes, but the steps described in the documentation do not work on my machine.
- [X] Yes, but I am having difficulty understanding it and want clarification.

**You are setting up traggo in**
- [X] Docker
- [ ] Linux native platform
- [ ] Windows native platform

**Describe your problem**
Hi, I'd like to know what to type to define a static range?
What is the correct way to specify a date?

I've tried to use the example in the doc and other date formats, but none of them seemed to work (see screenshot)

![image](https://user-images.githubusercontent.com/37513220/139135123-a3353e61-0eab-46f1-9d03-b3ffd3b0d389.png)

(from https://traggo.net/terminology/#static-range)

Thanks in advance!

**Any errors, logs, or other information that might help us identify your problem**

Ex: `docker-compose.yml`, `nginx.conf`, browser requests, etc.

![image](https://user-images.githubusercontent.com/37513220/139136228-51427724-b7e0-4fbe-b902-77047020ec52.png)

**Comment by lifo9** [comment]
@TheWiresharkGuy, I too struggled with this issue, the input really accepts only rfc3339 formated datetime - e.g.
**2021-11-02T00:00:00Z**

**Comment by langfingaz** [comment]
> Hey, this seems a little buggy because it is hardcoded and independent on the date locale. The format that should work is `YYYY-MM-DD HH:mm`. Example: 2021-10-10 14:11
> 
> So the time must be 24h based and PM / AM must be omitted.

It would be nice to add this to the [documentation](https://traggo.net/terminology/#static-range), once the UI of the front-end has been fixed.

**Comment by filcuk** [comment]
I'm on 0.2.3@f31968c1
Currently rfc3339 format shows as invalid both in UI and in the logs, as does any other date format I've tried:
- 2020-01-01T00:00:00Z
- 2020-01-01 00:00 AM
- 2020-01-01 00:00:00
- 2020-01-01 00:00
- 2020-01-01
- 2020/01/01 00:00
- 01/01/2020 00:00

Relative dates work just fine.

**Comment by pedrohba1** [comment]
Is it feasible to integrate a date picker UI? Just to make this process easier

**Comment by pedrohba1** [comment]
I would like to submit a datepicker feauter if possible, although I believe the easy path would be just using a datepicker module from material-ui or something else to give the back-end what it wants.

**Comment by jmattheis** [comment]
> Is it feasible to integrate a date picker UI? Just to make this process easier

Yes, but the first step would be to fix the bug. Afterward, we can think about improving the user experience.

**Comment by marksie1988** [comment]
Any news on when this will be fixed? I want to use Traggo but this kind of makes it useless for my needs. 

Doesn't look like there has been a release since April 2022?

**Comment by madotha** [comment]
I've stumbled across the same problem today, it's not working in:
- range in a dashboard item
- static range in dashboard
- editable range in dashboard

However a friend helped me out to at least get it working when having an editable time range in a dashboard, with the following format (same as discussed above, but also including milliseconds):

`2023-01-01T00:00:00.000Z`

Maybe this could help fixing the bug in the time parser?

**Comment by c-nv-s** [comment]
a date picker would be greatly appreciated once this is resolved
